config/bootscripts/boot-meson-gx.cmd

```diff
@@ -0,0 +1,75 @@
+# DO NOT EDIT THIS FILE
+#
+# Please edit /boot/armbianEnv.txt to set supported parameters
+#
+
+setenv overlay_error "false"
+# default values
+setenv rootdev "/dev/mmcblk1p1"
+setenv verbosity "1"
+setenv console "both"
+setenv bootlogo "false"
+setenv rootfstype "ext4"
+setenv docker_optimizations "on"
+
+# get PARTUUID of first partition on SD/eMMC it was loaded from
+# mmc 0 is always mapped to device u-boot (2016.09+) was loaded from
+if test "${devtype}" = "mmc"; then part uuid mmc ${devnum}:1 partuuid; fi
+if test "${devtype}" = "usb"; then part uuid usb ${devnum}:1 partuuid; fi
+echo "devtype: ${devtype}    ${partuuid}"
+if test "${console}" = "display"; then setenv consoleargs "console=tty1"; fi
+
+	# modern kernel boot
```

Comment:
Please, reformat file.

Reply:
Clarify please   Remove the single errant comment from the older file?

Reply:
*forgot this was on the LKML* 😉

Will do tonight.  This needs merged ASAP since it only has to exist because our "too many cooks" family model broke a supporting vendors board and can't be fixed without breaking other people's hacks.

Reply:
updated